File: add/codesnippet/VisualBasic/21e7f799-357a-4596-847b-_1.vb
        ' Override the OnAttributeRender method to
        ' not render the bgcolor attribute, which is 
        ' not supported in CHTML.
        Protected Overrides Function OnAttributeRender(ByVal name As String, ByVal value As String, ByVal key As HtmlTextWriterAttribute) As Boolean
            If (String.Equals("bgcolor", name)) Then
                Return False
            End If

            ' Call the ChtmlTextWriter version of 
            ' the OnAttributeRender method.
            MyBase.OnAttributeRender(name, value, key)

        End Function

File: add/codesnippet/CSharp/21e7f799-357a-4596-847b-_1.cs
		// Override the OnAttributeRender method to
		// not render the bgcolor attribute, which is
		// not supported in CHTML.
		protected override bool OnAttributeRender(string name, string value, HtmlTextWriterAttribute key)
		{
			if (String.Equals("bgcolor", name))
			{
				return false;
			}
			
			// Call the ChtmlTextWriter version of the
			// the OnAttributeRender method.
			return base.OnAttributeRender(name, value, key);
		}